code-challenges.test.js

```diff
@@ -68,6 +170,18 @@ var middleLetters2 = "rhinoceros"
 
 
 // b) Create the function that makes the test pass.
+//create a function called findMiddleOfWord
+//takes in a string as an argument
+//the function will determine weather the number of character in the string are odd or even
+//if the number is even the function will return the middle two numbers
+//else the number would have to be odd therefore it would return the middle character
+
+const findMiddleOfWord = (str) => {
+    //substring is used to to return the two middle letters by using math ceiling which rounds up to the next letter then subtracts one
+    return str.substr(Math.ceil(str.length / 2 - 1), //<---- if true half rounded up minus one will be returned
```

Comment:
```suggestion
    return str.substr(Math.ceil(str.length), //<---- if true half rounded up minus one will be returned
```
because `2-1` is always 1 and anything divided by itself is "itself"